src/components/DisplayEntries.tsx

```diff
@@ -1,15 +1,23 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { Card, Col, MarginTop, CardType, Padding, HorizontalAlignment } from '@skillsets/react-components';
-import { any } from 'prop-types';
 import { AppState } from '../reducers';
+import { Employee } from '../reducers/employeeReducer';
 
-interface DisplayEntriesProps {
-    salaries: number[];
+
+interface DisplayEntriesOwnProps {}
+
+interface DisplayEntriesStateProps {
+    employees: Array<Employee>;
```

Comment:
We typically use the syntax `Employee[]` rather than `Array<Employee>`, but it's just semantic